packages/beacon-node/src/chain/prepareNextSlot.ts

```diff
@@ -196,6 +196,10 @@ export class PrepareNextSlotScheduler {
             proposerIndex,
             feeRecipient,
           });
+
+          this.chain.executionEngine.getClientVersion().catch((e) => {
+            this.logger.error("Unable to get client version", {caller: "prepareNextSlot"}, e);
```

Comment:
I am also wondering if we need to call this here at all because we should be updating it any time the EL comes online already, I don't really see a scenario where this would be out of date